pyiron_base/jobs/dynamic.py

```diff
@@ -1,135 +1,16 @@
-        )
-        self.to_hdf()
-
-    return collect_output
-
-
-def _get_script(cp):
-    spec = importlib.util.spec_from_file_location(
-        "script", os.path.join(cp, "script.py")
-    )
-    script = importlib.util.module_from_spec(spec)
-    spec.loader.exec_module(script)
-    return script
-
-
-def _init_constructor_dynamic(class_name, module, input_dict):
-    from pyiron_base.jobs.job.extension.executable import Executable
-
-    def __init__(self, project, job_name):
-        super(self.__class__, self).__init__(project, job_name)
-        self.__name__ = class_name
-        self.input.update(input_dict)
-        self._executable = Executable(
-            codename=class_name,
-            module=module,
-            path_binary_codes=None,
-        )
-
-    return __init__
-
-
-def class_constructor(cp):
-    if "templates" in cp:
-        from pyiron_base.jobs.script import ScriptJob
-
-        class_name = os.path.basename(cp)
-        script_path = os.path.join(cp, "script.py")
-        input_dict = _load_input(path=cp)
-        return type(
-            class_name,
-            (ScriptJob,),
-            {
-                "__init__": _init_constructor_script(
-                    class_name=class_name,
-                    script_path=script_path,
-                    input_dict=input_dict,
-                )
-            },
-        )
-    elif "dynamic" in cp:
-        from pyiron_base.jobs.job.template import TemplateJob
-
-        class_name = os.path.basename(cp)
-        input_dict = _load_input(path=cp)
-        script = _get_script(cp=cp)
-        return type(
-            class_name,
-            (TemplateJob,),
-            {
-                "__init__": _init_constructor_dynamic(
-                    class_name=class_name,
-                    module=os.path.join(
-                        os.path.basename(os.path.dirname(cp)), os.path.basename(cp)
-                    ),
-                    input_dict=input_dict,
-                ),
-                "write_input": _get_write_input(script=script),
-                "collect_output": _get_collect_output(script=script),
-            },
-        )
-
-
-def _get_template_classes():
-    return {
-        os.path.basename(cp): cp
-        for cp in _get_class_path_lst(prefix="templates")
-        + _get_class_path_lst(prefix="dynamic")
-    }
-
-
-JOB_DYN_DICT = _get_template_classes()
+def warn_dynamic_job_classes(resource_folder_lst, logger):
+    for path in resource_folder_lst:
+        if os.path.exists(path):
+            sub_folders = os.listdir(path)
+            for folder in ["dynamic", "templates"]:
+                if folder in sub_folders:
+                    logger.warning(
+                        "pyiron found a '"
+                        + folder
+                        + "' folder in the "
+                        + path
+                        + " resource directory. These are no longer supported in pyiron_base >=0.7.0."
```

Comment:
Would it be possible to point to the new feature here as well? (Is it actually merged, yet? Otherwise, I would like to see some pointer to the feature there)